src/components/Audit.vue

```diff
@@ -0,0 +1,253 @@
+          <div class="linear1"></div>
+          <div class="column1 column">
+            <p>{{ audit.id }}</p>
+          </div>
+          <div class="column2 column">
+            <p>{{ audit.cause}}</p>
+          </div>
+          <div class="column3 column">
+            <p>{{ audit.consequence}}</p>
+          </div>
+        </li>
+      </ul>
+    </div>
+
+    <div id="footer">
+      <button id="footer-btn" @click="toggleMenu" v-click-outside="hideMenu">{{ this.rowSize }} Rows</button>
+      <div id="menu" v-show="menu">
+        <div
+          class="menu-article"
+          v-for="rowValue in rowSizesValue"
+          :key="rowValue"
+          @click="setRows(rowValue)"
+        >{{ rowValue }}</div>
+      </div>
+
+      <div class="text-xs-center">
+        <v-pagination v-model="page" :length="pagesSize" :total-visible="5" @input="changePage"></v-pagination>
+      </div>
+    </div>
+
+    <div
+      id="notification"
+      v-show="showNoti"
+      :class="{redBorder: errorOccured, greenBorder: !errorOccured}"
+    >
+      <input
+        type="text"
+        v-model="textNoti"
+        readonly
+        :class="{redText: errorOccured, greenText: !errorOccured}"
+      />
+      <button
+        @click="showNotification"
+        :class="{redBackground: errorOccured, greenBackground: !errorOccured}"
+      >OK</button>
+    </div>
+  </div>
+</template>
+
+<script>
+import { API_BASE_URL } from "../constants/index.js";
+import { Current_User_Role, THEME_ID, THEME } from "../constants/index.js";
+import { User_Email } from "../constants/index.js";
+import { ACCESS_TOKEN, CAUSE ,CONSEQUENCE ,USER_LANGUAGE, ACTIVITY_LOG } from "../constants/index.js";
+import axios from "axios";
+import ClickOutside from "vue-click-outside";
+const headers = {
+  "Content-Type": "application/json",
+  Authorization: "Bearer " + localStorage.getItem(ACCESS_TOKEN)
+};
+
+export default {
+  name: "Audit",
+  data(){
+      return{
+        auditData: [],
+        textNoti: "",
+        showNoti: false,
+        rowSizesValue: [5, 10, 20],
+        rowSize: 20,
+        page: 1, //current active page
+        pagesSize: 1, //Number of pages
+        menu: false,
+        sortByValue: "createdAt",
+        textNoti: "",
+        errorOccured: false,
+        showNoti: false
+      }
+  },
+  mounted: function(){
+     if (localStorage.getItem(THEME) == "light") {
+      document.getElementById("header").style.backgroundColor = "white";
+      document.getElementById("divlist").style.backgroundColor = "white";
+    } else if (localStorage.getItem(THEME) == "dark") {
+      document.getElementById("header").style.backgroundColor = "black";
+      document.getElementById("divlist").style.backgroundColor = "black";
+      document.getElementById("Audit").style.backgroundColor = "black";
+    }
+     if (localStorage.getItem(USER_LANGUAGE) != "en") {
+       document.getElementsByTagName("H1")[0].innerHTML = localStorage.getItem(
```

Comment:
Same as above - we should avoid direct html elements access. this is vue's job.

Consider keeping list of needed translations in some object (it can be also external service) and binding element value to needed value from object.

So, object would be:

```
translations: {
  ACTIVITY_LOG: "" // add translation here
}
```

and we would bind it with something like `<h1>{{ translations.ACTIVITY_LOG }}</h1>`

It might not fit application now, but consider this approach